src/tests/ftest/util/environment_utils.py

```diff
@@ -0,0 +1,560 @@
+"""
+  (C) Copyright 2018-2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import json
+import os
+import site
+
+from ClusterShell.NodeSet import NodeSet
+
+from network_utils import get_fastest_interface, get_common_provider, SUPPORTED_PROVIDERS, \
+    PROVIDER_ALIAS, NetworkException
+from run_utils import run_remote
+
+
+class TestEnvironmentException(Exception):
+    """Exception for launch.py execution."""
+
+
+def get_build_environment(logger, build_vars_file):
+    """Obtain DAOS build environment variables from the .build_vars.json file.
+
+    Args:
+        logger (Logger): logger for the messages produced by this method
+        build_vars_file (str): the full path to the DAOS build_vars.json file
+
+    Raises:
+        TestEnvironmentException: if there is an error obtaining the DAOS build environment
+
+    Returns:
+        dict: a dictionary of DAOS build environment variable names and values
+
+    """
+    logger.debug("Obtaining DAOS build environment PREFIX path from %s", build_vars_file)
+    try:
+        with open(build_vars_file, encoding="utf-8") as vars_file:
+            return json.load(vars_file)
+
+    except Exception as error:      # pylint: disable=broad-except
+        raise TestEnvironmentException("Error obtaining build environment:", str(error)) from error
+
+
+def update_path(logger, build_vars_file):
+    """Update the PATH environment variable for functional testing.
+
+    Args:
+        logger (Logger): logger for the messages produced by this method
+        build_vars_file (str): the full path to the DAOS build_vars.json file
+
+    Raises:
+        TestEnvironmentException: if there is an error obtaining the DAOS build environment
+    """
+    base_dir = get_build_environment(logger, build_vars_file)["PREFIX"]
+    bin_dir = os.path.join(base_dir, "bin")
+    sbin_dir = os.path.join(base_dir, "sbin")
+
+    # /usr/sbin is not setup on non-root user for CI nodes.
+    # SCM formatting tool mkfs.ext4 is located under /usr/sbin directory.
+    usr_sbin = os.path.join(os.sep, "usr", "sbin")
+    path = os.environ.get("PATH")
+
+    # Update PATH
+    os.environ["PATH"] = ":".join([bin_dir, sbin_dir, usr_sbin, path])
```

Comment:
What if PATH isn't empty?

Reply:
Included as part of `path = os.environ.get("PATH")`